pkg/apps/deploymentconfig/controller_ref_manager.go

```diff
@@ -91,14 +91,14 @@ func (m *RCControllerRefManager) ClaimReplicationController(rc *v1.ReplicationCo
 	match := func(obj kmetav1.Object) bool {
 		return m.Selector.Matches(klabels.Set(obj.GetLabels()))
 	}
-	adopt := func(obj kmetav1.Object) error {
+	adopt := func(ctx context.Context, obj kmetav1.Object) error {
 		return m.AdoptReplicationController(obj.(*v1.ReplicationController))
 	}
-	release := func(obj kmetav1.Object) error {
+	release := func(ctx context.Context, obj kmetav1.Object) error {
 		return m.ReleaseReplicationController(obj.(*v1.ReplicationController))
 	}
-
-	return m.ClaimObject(rc, match, adopt, release)
+	ctx := context.TODO()
```

Comment:
/cc @soltysh 

There are a few items here which require `context` arguments. For now I'm adding TODO contexts, but in the future I'd like to wire the main context all the way down here.